libs/filabridge/include/filament/MaterialEnums.h

```diff
@@ -28,7 +28,7 @@
 namespace filament {
 
 // update this when a new version of filament wouldn't work with older materials
-static constexpr size_t MATERIAL_VERSION = 42;
+static constexpr size_t MATERIAL_VERSION = 43;
```

Comment:
You also need to add an entry to `NEW_RELEASE_NOTES.md` and specify  [⚠️ **New Material Version**].
It is actually not required to update the version here because the person who does the weekly release, will do it based on the release notes.